pkg/config/config_test.go

```diff
@@ -157,6 +157,14 @@ fDGt+yaf3RaZbVwHSVLzxiXGsu1WQJde3uJeNh5c6z+5
 			},
 			"*backends.LocalSecretManager",
 		},
+		{
+			map[string]interface{}{
+				"AVP_TYPE":             "1passwordconnect",
+				"AVP_OP_CONNECT_TOKEN": "token",
+				"AVP_OP_CONNECT_HOST":  "opconnect.supergrain.com",
```

Comment:
should we use a generic name here? do you intend to submit this as pull request to parent repo?